backend/src/test/kotlin/pl/akai/fillist/web/spotifywrapper/SpotifyPlaylistsTests.kt

```diff
@@ -52,4 +52,18 @@ class SpotifyPlaylistsTests {
         // assertEquals(playlist.description, createPlaylistRequestBody.description)
         assertEquals(playlist.owner.id, userId)
     }
+
+    @Test
+    fun getPlaylist() {
+        // Sample playlist ID
```

Comment:
Comments quickly become outdated, making them less helpful for understanding code. They can also make code more difficult to understand, especially if they are not updated as the code changes. If you look at the code below, you know what that stuff makes. In my opinion these comment is useless.

Reply:
Agree, I removed redundant comment.